### ISSUE TITLE
Fix Danger

### ISSUE THREAD

**Issue by gregheo** [SwiftWeekly/swiftweekly.github.io#284]
Is it the recent pod upgrade?

Prose lint always seemed a bit flaky with its suggestions and super noisy to me.

## Danger has errored
[!] Invalid `Dangerfile` file: wrong number of arguments (2 for 1)
```
 #  from Dangerfile:2
 #  -------------------------------------------
 #  # Look for prose issues
 >  prose.lint_files
 #  
 #  -------------------------------------------
```

**Comment by BasThomas** [comment]
I've opened https://github.com/dbgrandi/danger-prose/issues/20

**Comment by dbgrandi** [comment]
It *should* be fixed in danger-prose 2.0.5...just published.

**Comment by BasThomas** [comment]
Thanks for the quick resolution, @dbgrandi 😃. Let's see if this works.
